Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md

```diff
@@ -1,2 +1 @@
 Deploy this template to create Logic APP for notifying us on new IDPS signature updates.
```

Comment:
Please reword:

_Deploy this template to create a Logic App to receive notifications on new IDPS signature updates for Azure firewall_

Is the Deploy button implemented yet? if not, we cannot use the word "Deploy"

Reply:
The deploy button is not created yet - For now we can say  -  "Use this template"

Reply:
Made the requested change